packages/config/config/devices/0x0131/ne_nas_ab02z.json

```diff
@@ -0,0 +1,160 @@
+// Zipato NE-NAS-AB02Z
+// Indoor Siren
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "NE-NAS-AB02Z",
+	"description": "Indoor Siren",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x1083",
+			"zwaveAllianceId": 3046
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Siren status",
+			"description": "is assigned to the siren status - ON/OFF. Allows for sending a controlled command to the associated device whenever the Indoor siren is turned ON (or OFF).",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Configure alarm volume",
+			"description": "Low, Medium, High",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Low",
+					"value": 1
+				},
+				{
+					"label": "Medium",
+					"value": 2
+				},
+				{
+					"label": "High",
+					"value": 3
+				}
+			]
+		},
+		"2": {
+			"label": "Configure alarm duration time",
+			"description": "30 sec, 1 min, 5 min",
+			"valueSize": 1,
+			"minValue": 0,
```

Comment:
1?

Reply:
That’s the default in the file. I’m not sure that it’s unusual you’d want a siren to go off for a while if activated. Do you prefer a different default?